spec/javascripts/health-care/components/personal-information/VeteranAddressSection.spec.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import SkinDeep from 'skin-deep';
+
+import VeteranAddressSection from '../../../../../_health-care/_js/components/personal-information/VeteranAddressSection';
+
+describe('<VeteranAddressSection>', () => {
+  const nullAddress = [{
+    street: null,
+    city: null,
+    country: null,
+    state: null,
+    zipcode: null,
+  }];
+
+  it('Sanity check the component renders', () => {
+    const tree = SkinDeep.shallowRender(<VeteranAddressSection data={{ address: nullAddress }}/>);
+    const vdom = tree.getRenderOutput();
+    expect(vdom).to.have.property('type', 'fieldset');
```

Comment:
Right, what I was trying to say is that the wrapper for the component varies from component to component based on the particular markup and what is needed from a styles perspective. My concern with this test in verifying whether the component rendered based on the type of the wrapper is that: 1) the wrapper is not really representative of what a section should contain, 2) it's not consistent and could easily change if the markup changes. I was just wondering if it would be better to verifying render based on some other attribute, something we know is consistent about a section. `fieldset` carries with it particular form styling from the U.S. Web Design Standards, so that's why we need to include it as a wrapper.